webapp/src/components/admin_settings/jaas_section.tsx

```diff
@@ -1,5 +1,7 @@
 import * as React from 'react';
-import {FormattedMessage} from 'react-intl';
+
+import {InputField} from '../InputField/InputField';
+import {RadioField} from '../RadioField/RadioField';
```

Comment:
rename these files to `index.tsx` then u can specify the path as `../InputField`......that looks neater than writing `../InputField/InputFiel`